lib/jsonapi/plugs/format_required.ex

```diff
@@ -5,12 +5,20 @@ defmodule JSONAPI.FormatRequired do
 
   import JSONAPI.ErrorView
 
+  # Cf. https://jsonapi.org/format/#crud-updating-to-many-relationships
+  @update_has_many_relationships_methods ~w[DELETE PATCH POST]
```

Comment:
Technically, DELETE is handled earlier by the first clause. But it is technically among the allowed methods for the multi-RIO payload, so I thought including it here made more sense than omitting it.